examples/apps/generic_client/generic_client.py

```diff
@@ -178,6 +180,33 @@ def _lookup_first_worker(worker_registry, jrpc_req_id):
     return worker_id
 
 
+def _do_IAS_verification(worker_obj):
```

Comment:
Can we rename to more generic as `_do_worker_verfication()`. Avalon clients is agnostic about the details of how attestation is happening at back end (IAS or DCAP).  Only Avalon core framework will have the visibility of details of attestation.